server/routes/templates.ts

```diff
@@ -15,49 +16,47 @@ import {
 	onlyAllowAnonymousBranch, branchRedirector, ApplicationType
 } from "../middleware";
 import {
-	IUser, IUserMongoose, User,
-	ITeamMongoose, Team,
-	IIndexTemplate, ILoginTemplate, IAdminTemplate, ITeamTemplate,
+	Model,
+	IUser, User,
+	ITeam, Team,
+	IIndexTemplate, IAdminTemplate, ITeamTemplate,
 	IRegisterBranchChoiceTemplate, IRegisterTemplate, StatisticEntry,
 	IFormItem,
-	IConfig
+	ILoginTemplate
 } from "../schema";
 import * as Branches from "../branch";
-import { strategies, prettyNames } from "../routes/strategies";
 
 export let templateRoutes = express.Router();
 
-// Load and compile Handlebars templates
-let [
-	indexTemplate,
-	loginTemplate,
```

Comment:
is there some feature that automatically makes a class variable? How does `loadTemplate` get `this.file`?

Reply:
The constructor of the `Template` class has `constructor(private readonly file: string) {}` which makes the first argument when called with `new` a `private readonly` property automatically. This is shorthand supported by TypeScript

Reply:
neat